standard/statements.md

```diff
@@ -676,9 +676,9 @@ The ***governing type*** of a `switch` statement is established by the switch ex
 
 There can be at most one `default` label in a `switch` statement.
 
-It is an error if the pattern of any switch label is not *applicable* (NEED AN XREF) to the type of the input expression.
+It is an error if the pattern of any switch label is not *applicable* (§pattern-forms-general) to the type of the input expression.
 
-It is an error if the pattern of any switch label is *subsumed* by (NEED AN XREF) the set of patterns of earlier switch labels of the switch statement that do not have a case guard or whose case guard is a constant expression with the value true.
+It is an error if the pattern of any switch label is *subsumed* by ([§11.3]patterns.md#113-pattern-subsumption) the set of patterns of earlier switch labels of the switch statement that do not have a case guard or whose case guard is a constant expression with the value true.
```

Comment:
This is causing the error in the renumbering tool. The link is malformed. I just removed the MD link syntax so the tool will add it when it runs:

```suggestion
It is an error if the pattern of any switch label is *subsumed* by (§11.3) the set of patterns of earlier switch labels of the switch statement that do not have a case guard or whose case guard is a constant expression with the value true.
```